### ISSUE TITLE
Recent Medical Center Staffing Data

### ISSUE THREAD

**Issue by momiji15** [department-of-veterans-affairs/va-data#23]
Hello,

I am wondering if you have recent medical center staffing data for VA Health facilities. The most recent dataset I could find was the Veterans Health Administration 2010 Facility Quality and Safety Report - Medical Center Staffing dataset.

When I took at a look at the dataset, it does not include other health facilities such as Community Based Outpatient Clinics and Health Care Centers. Is there a dataset that includes staffing for those facilities?

**Comment by VHA-NDS-2** [comment]
Good Afternoon,

Thank you for your inquiry regarding Medical Center Staffing data.  Unfortunately VA does not have CBOC facility staffing level data that is publicly reported.  Obtaining this information would require a special data pull from the Primary Care program office.  If you wish to obtain this information, please submit a FOIA request.    

FOIA Request:
VA FOIA Website:  https://www.oprm.va.gov/foia/
FOIA Request:  https://www.oprm.va.gov/foia/howto_file_foia_request.aspx

Office Reporting, Analytics, Performance, Improvement and Deployment (RAPID) 10EA
Veterans Health Administration (VHA)

**Comment by momiji15** [comment]
Thank you for your reply! I will go ahead and do that. Is there more recent staffing data for VAMCs other than the 2010 Quality and Safety Report?

**Comment by VHA-NDS-2** [comment]
Yes, there is the 2012 Quality and Safety Report (https://www.va.gov/HEALTH/docs/VHA_Quality_and_Safety_Report_2013.pdf).  The report is no longer published; therefore, this is the final version of the report.

Office Reporting, Analytics, Performance, Improvement and Deployment (RAPID) 10EA
Veterans Health Administration (VHA)
